src/FxKit/Option/Option.Do.cs

```diff
@@ -22,4 +22,18 @@ public static Option<T> Do<T>(this Option<T> source, Action<T> action)
 
         return source;
     }
+
+    /// <summary>
+    /// Asynchronously runs <see cref="callback" /> if the <paramref name="source" /> is Some.
+    /// </summary>
+    public static async Task<Option<T>> DoAsync<T>(this Option<T> source, Func<T, Task> callback)
```

Comment:
```suggestion
    [GenerateTransformer]
    public static async Task<Option<T>> DoAsync<T>(this Option<T> source, Func<T, Task> callback)
```

Reply:
I'll merge this as-is and add the `[GenerateTransformer]` in my other PR.

Reply:
Sorry the delay